freq/views.py

```diff
@@ -93,6 +95,10 @@
         'additive': 'disabled',
         'frequency': 'disabled'
     },
+    'download': {
+        'organisations': {},
```

Comment:
Ik weet niet of je deze dict van dicts van dicts in je python code gebruikt of alleen rechtstreeks via json opstuurt, maar.... Een `copy.deepcopy()` is wel aan te raden als je er *iets* mee doet in python. Anders vervuil je hem snel ook voor andere threads.